Add support for distinct trim strategies (#62)
uphold/anonymizer: README.md

    `options.serializers` _(List[Object])_: A list with serializers to apply. Each serializers must contain two properties: `path` (path for the value to be serialized, must be a `string`) and `serializer` (function to be called on the path's value).

>     `options.strategy` _(string)_: A flag that specifies the trimming strategy. Should be one of the following:

Reviewer: ```suggestion
    `options.strategy` _(string)_: An option that specifies the trimming strategy. Should be one of the following:
```

Author: Done

>      * `redact` is the default behavior and obfuscates the value without trimming.

Author: Done

     * `trim` removes the redacted values from the output.
     * `trim-and-list` removes and adds a list with all deleted paths.

> ### Example

Reviewer 2: Maybe complete de description?

```
#### Example using `redact` as strategy
```

Author: Done
